lms/djangoapps/hera/fragments.py

```diff
@@ -1,20 +1,17 @@
-from opaque_keys.edx.keys import CourseKey
-from django.template.loader import render_to_string
 from django.template.context_processors import csrf
-
+from django.template.loader import render_to_string
+from opaque_keys.edx.keys import CourseKey
 from web_fragments.fragment import Fragment
 
-from courseware.courses import get_course_overview_with_access
+from courseware.courses import get_course_overview_with_access, get_current_child
 from courseware.model_data import FieldDataCache
-from courseware.module_render import  get_module_for_descriptor
-from courseware.courses import get_current_child, get_problems_in_section
-
-from openedx.features.course_experience.views.course_outline import CourseOutlineFragmentView
+from courseware.module_render import get_module_for_descriptor
 from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
-
+from lms.djangoapps.hera.mongo import BLOCK_ID, COURSE_KEY, LEVEL, NEXT_ID, USER, COMPLETED_COURSES, c_selection_page
+from openedx.features.course_experience.utils import get_course_outline_block_tree
+from openedx.features.course_experience.views.course_outline import CourseOutlineFragmentView
 from xmodule.modulestore.django import modulestore
-from openedx.features.course_experience.utils import get_course_outline_block_tree, get_resume_block
-from lms.djangoapps.hera.mongo import c_selection_page, USER, COURSE_KEY, BLOCK_ID, NEXT_ID, LEVEL
+
 
 def unit_grade_level(earned, course_block_tree, subsection_active, request, course_id=None):
```

Comment:
please rename this `subsection_active` to `last_visited_subsection` to be more clear.